index.html

```diff
@@ -271,13 +310,14 @@ <h3>Used in</h3>
         const app = new Vue({
             el: '#app',
             data: {
-                data: { index: {}, costs: {}, data: {} },
+                data: { index: {}, costs: {}, data: {}, recipeCount: 0 },
                 currentElement: "",
                 showHiddenRecipes: false,
                 searchQuery: "",
                 showToast: true,
                 saveData: [],
-                loaded: 1000
+                loaded: 1000,
+                slideTransition: "slide-right",
```

Comment:
I like the transitions, but try to avoid adding state here just for styling. Can this be computed or a method?

Reply:
As far as I can tell, doing it this way is the most consistent method if we want to have multiple transitions that can happen on the same elements. There may be some potential work-arounds but I haven't found anything other than this in the documentation